src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/LegacyPbaResponse.java

```diff
@@ -9,10 +9,10 @@
 
     @JsonProperty
     @SuppressWarnings({"checkstyle:MemberName"})
-    private final List<String> payment_accounts;
+    private final List<String> paymentAccounts;

```

Comment:
it is good to follow Sonar comment here for not having variable name in camel context , but since FR is already consuming the response field "payment_accounts" and they will fail to get response field, so revert back and make it as existing.